cove/templatetags/cove_tags.py

```diff
@@ -56,3 +62,87 @@ def take_or_sample(population, k):
 def list_from_attribute(list_of_dicts, key_name):
     return [value[key_name] for value in list_of_dicts]
 
+
+
+# These are "safe" html that we trust
+# Don't insert any values into these strings without ensuring escaping
+# e.g. using django's format_html function.
+validation_error_template_lookup_safe = {
+    "date-time": _("Date is not in the correct format"),
+    "uri": _("Invalid 'uri' found"),
+    "string": _("{}<code>{}</code> is not a string. Check that the value {} has quotes at the start and end. Escape any quotes in the value with <code>\</code>"),
+    "integer": _("{}<code>{}</code> is not a integer. Check that the value {} doesn’t contain decimal points or any characters other than 0-9. Integer values should not be in quotes. "),
+    "number": _("{}<code>{}</code> is not a number. Check that the value {} doesn’t contain any characters other than 0-9 and dot (<code>.</code>). Number values should not be in quotes. "),
+    "object": _("{}<code>{}</code> is not a JSON object"),
+    "array": _("{}<code>{}</code> is not a JSON array"),
+}
+
+
+@register.filter(name='html_error_msg')
+def html_error_msg(error):
+
```

Comment:
Might be nice to have a doc string description here given that it is a library function.